samples/IdentityServerSample/Controllers/AccountController.cs

```diff
@@ -67,7 +67,7 @@ public async Task<IActionResult> ExternalLoginCallback()
 
             if (_interaction.IsValidReturnUrl(returnUrl) || Url.IsLocalUrl(returnUrl))
             {
-                return Redirect(returnUrl);
+                return LocalRedirect(returnUrl);
```

Comment:
Might be a mistake, we do not take all the time returnUrl as a local one, first value in the condition does not guaranty that
```suggestion
                return Redirect(returnUrl);
```